product_docs/docs/pgd/5/postgres-configuration.mdx

```diff
@@ -51,10 +51,10 @@ N slots and WAL senders. During synchronization, PGD temporarily uses another
 N-1 slots and WAL senders, so be careful to set the parameters high enough
 for this occasional peak demand.
 
-With parallel apply turned on, the number of slots must be increased to
-N slots from the formula \* writers. This is because the `max_replication_slots`
+With Parallel Apply turned on, the number of slots must be increased to
+N slots from the formula \* writers. This is because `max_replication_slots`
 also sets the maximum number of replication origins, and some of the functionality
-of parallel apply uses extra origin per writer.
+of Parallel Apply uses extra origin per writer.
```

Comment:
Is this missing the word "an," i.e., "an extra origin per writer"?

Reply:
yes, an extra,